### DISCUSSION TITLE
Example usage in Docker Swarm

### DISCUSSION THREAD

**Discussion by jmb12686** [crazy-max/docker-cloudflared#97] [Q&A] [answered]
@crazy-max , thank you for providing this multi-arch image.  I got it working with pihole in my Docker Swarm cluster.  If you wanted to provide examples in your repo (along with your docker-compose.yml), please see my[ swarm compose](https://github.com/jmb12686/raspi-docker-stacks/blob/master/pihole/docker-compose-stack.yml) and [README](https://github.com/jmb12686/raspi-docker-stacks/blob/master/pihole/README.md)

**Comment by crazy-max** [comment, marked as answer]
Hi @jmb12686

That's awesome thanks. Feel free to open a PR :)
You can take a look at my [docker-matomo repo](https://github.com/crazy-max/docker-matomo#use-this-image) as a layout example to include swarm.
